fdbserver/RESTKmsConnector.actor.cpp

```diff
@@ -791,9 +796,9 @@ Future<T> kmsRequestImpl(
 				ctx->kmsUrlStore.penalize(*urlCtx, KmsUrlCtx<KmsUrlPenaltyParams>::PenaltyType::TIMEOUT);
 				// Keep re-trying if KMS request time-out OR is server unreachable; otherwise, bubble up the error
 				if (!isKmsNotReachable(e.code())) {
-					TraceEvent(SevDebug, "KmsRequestFailedUnreachable", ctx->uid)
-					    .error(e)
-					    .detail("RequestID", requestID);
+					if (FLOW_KNOBS->REST_LOG_LEVEL >= RESTLogSeverity::DEBUG) {
+						TraceEvent("KmsRequestFailedUnreachable", ctx->uid).error(e).detail("RequestID", requestID);
+					}
 					throw e;
 				}
 				TraceEvent(SevDebug, "KmsRequestError", ctx->uid).error(e).detail("RequestID", requestID);
```

Comment:
Can we also guard this by `FLOW_KNOBS->REST_LOG_LEVEL >= RESTLogSeverity::DEBUG` and make it `SevInfo`

Reply:
I think given it is error handling, it can exists outside of the knob and also it helps look at requestID based logging, will leave it as is for now.

Reply:
It's `SevDebug` so it won't show up by default anyways, might be better to then change it to `SevInfo` or `SevWarn`

Reply:
It used to be `SevWarn` and idea is to avoid log-spew as it is not a fatal log and mostly used for debugging. 
I can guard it under REST_LOG_LEVEL if we think that is helpful as a followup PR.